src/aws-api-mcp-server/awslabs/aws_api_mcp_server/core/aws/service.py

```diff
@@ -175,6 +179,14 @@ def interpret_command(
     )
 
 
+def _log_successful_execution(response: InterpretationResponse | None):
```

Comment:
Why do we have to log executions in the first place? I'm missing the details here.